docs/_assets/attributes.adoc

```diff
@@ -0,0 +1,38 @@
+// AsciiDoc settings
+:data-uri!:
+:doctype: book
+:experimental:
+:idprefix:
+:imagesdir: images
+:numbered:
+:sectanchors!:
+:sectnums:
+:source-highlighter: highlight.js
```

Comment:
This switches from `pygments` to `highlight.js` is that intentional / valid here upstream?